index.html

```diff
@@ -0,0 +1,152 @@
+            </div>
+        <div class="container">
+            <div class="imgdiv">
+                <img class="projects" src="image1.png" alt="Nature">
+            </div>
+            <div class="tonic">
+                    <h1 class="popheading">Tonic</h1>
+                    <p class="popp13">CANOPY <img class="dot" src="dot.png" alt="dot"> 
+                        <span class="popp13 col">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <span class="htcj">HTML</span> 
+                    <span class="htcj">CSS</span>
+                    <span class="htcj">Javascript</span><br><br>
+                      <div class="btn">
+                        <a href="image1.png" target="_blank">See Project</a>
+                      </div>
+            </div>    
+           </div>
+           <div class="container2">
+            <div class="imgdiv2">
+                <img class="projects" src="image2.png" alt="Nature">
+            </div>
+            <div class="multi">
+                    <h1 class="popheading">Multi Post Series</h1>
+                    <p class="popp13">CANOPY <img class="dot" src="dot.png" alt="dot"> 
+                        <span class="popp13 col">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <br>
+                    <span class="htcj2">HTML</span> 
+                    <span class="htcj2">CSS</span>
+                    <span class="htcj2">Javascript</span><br><br>
+                      <div class="btn2">
+                        <a href="image2.png" target="_blank">See Project</a>
+                      </div> 
+                      </div>
+            </div>    
+           </div>
+           <div class="container">
+            <div class="imgdiv">
+                <img class="projects" src="image3.png" alt="Nature">
+            </div>
+            <div class="tonic">
+                    <h1 class="popheading">Tonic</h1>
+                    <p class="popp13">CANOPY <img class="dot" src="dot.png" alt="dot"> 
+                        <span class="popp13 col">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <span class="htcj">HTML</span> 
+                    <span class="htcj">CSS</span>
+                    <span class="htcj">Javascript</span><br><br>
```

Comment:
- [ ] [REQUIRED] Please implement the [previous reviewer's](https://github.com/baqar-abbas/Portfolio-Project-Final-version/pull/1/files#r1073049510) suggestion 🙏🏼 The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should be a list `ul`, and each of them a `li`, not a `span` 😃